addon/ng2/blueprints/ng2/files/config/karma.conf.js

```diff
@@ -22,10 +23,34 @@ module.exports = function (config) {
       { pattern: './src/test.ts', watched: false }
     ],
     preprocessors: {
+      './src/**/**/*.ts': ['angular-cli-coverage'],
```

Comment:
I added a new preprocessor in the karma plugin. While there is a single entry point for the tests in order to get the remap plugin to generate the mapped coverage report it needs to  run over the test files themselves.